testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -961,3 +961,11 @@ def test_partially_missing_cryst():
     assert len(u.atoms) == 3
     assert len(u.trajectory) == 2
     assert_array_almost_equal(u.dimensions, 0.0)
+
+def test_cryst_em_warning():
+    #issue 2599  
+    with pytest.warns(UserWarning) as record:
+        u = mda.Universe(PDB_CRYOEM_BOX)
+        cur_sele = u.select_atoms('around 0.1 (resid 4 and name CA and segid A)') 
```

Comment:
As per my previous comment, I don't think you need to have a `select_atoms` call here to trigger the warning.

Instead I would create a second test that does an atom selection on `PDB_CRYOEM_BOX` that has this check.